helmfile.d/20-s3.yaml

```diff
@@ -52,6 +52,8 @@ releases:
     chart: radar/radar-s3-connector
     version: {{ .Values.radar_s3_connector._chart_version }}
     installed: {{ .Values.radar_s3_connector._install }}
+    needs:
+    - minio
```

Comment:
What happens if `minio.installed: false` when a proper S3 bucket is used?

Reply:
I haven't tested this yet but my assumption is that it won't install the app unless `--skip-needs` is used. I think we have to use something like `needs: {{- if minio._installed: true }}[minio]{{- end }}` here.

Reply:
After testing looks like if one of the dependencies isn't going to be installed then the chart that is dependent on it won't be installed either.